governance/tsc/tsc-2021-10-08.md

```diff
@@ -3,14 +3,48 @@
 Date: 2021-10-08
 Location: Zoom
 
+attending: asajeffrey,  jdm, cybai, simonsapin, avadacatavra
+
 ## Agenda
 
 * Approve 2021-06 meeting minutes
 
+approved.
+
 * Servo infrastructure
 
+- jdm: all travis CI stopped working, repositories need to be converted to GitHub or stop relying on homu.
+- jdm: deploying homu updates is annoying; saltfs is based on travis, keeps requiring updating to newer versions.
+- discussion of whether moving to bars.tech instead of self-hosted homu would allow turning off EC2 instance
```

Comment:
Found a typo here 👀 

```suggestion
- discussion of whether moving to bors.tech instead of self-hosted homu would allow turning off EC2 instance
```